docs/proposal-optional-output-extension.md

```diff
@@ -62,14 +62,36 @@ There are currently three problems with expiry (as of cylc-flow 8.1.4):
    )
    ```
 
-   Note, a task is *in*complete, if the completion condition is not satisfied
-   but the task has a final task status (note status not output). This is the
-   same as presently implemented (8.2.x)
+   Similarly, a task's outputs are "incomplete" or "not complete" if the completion
+   condition is not satisfied.
+
+3.5 A task is "finished" if it has final task status
```

Comment:
"Finished" already has a well established meaning (succeeded or failed) so I don't think we should use that. I suggest we stick with "final status" (as you've done in 3.6 below) or "final task status".

Reply:
Well, succeed and failed are the two primary final statuses.  But I agree the old `:finished` pseudo output is a spanner in those works.

So ok, "final status" it is.